src/components/fieldset/address-group/index.njk

```diff
@@ -18,7 +18,7 @@ stylesheets:
 
   {{ govukInput({
     label: {
-      html: 'Building and street <span class="govuk-visually-hidden">line 1 of 2</span>'
+      html: 'Address line 1'
```

Comment:
```suggestion
      text: 'Address line 1'
```

Reply:
I think if I'm being pedantic these should probably use 'text' rather than 'html' – we tend to use 'html 'when we need to include different html elements in the same place.

Reply:
This is a good catch – we want to encourage users to use `text` wherever possible (it's more secure) so we should make this change, and go back and change it in the examples in the address pattern as well.